integration-tests/src/test/resources/setupenv.sh

```diff
@@ -56,6 +56,9 @@ function create_image_pull_secret_wl {
 	  # below docker pull is needed to for domain home in image tests the base image should be in local repo
 	  docker login -u $OCR_USERNAME -p $OCR_PASSWORD ${OCR_SERVER}
 	  docker pull $IMAGE_NAME_WEBLOGIC:$IMAGE_TAG_WEBLOGIC
+	  
+	  # pull fmw infra images
+          docker pull $IMAGE_NAME_FMWINFRA:$IMAGE_TAG_FMWINFRA
```

Comment:
you may want to delete this here and add the below outside of the conditions at line number 139 so that it applies to all three envs and the images are pulled only when needed WLS or JRF runs.

```
  if [ "$JRF_ENABLED" = true ] ; then
	pull_tag_images_jrf	
  else
  	create_image_pull_secret_wl
  fi
```

Reply:
Done